Content.Shared/_RMC14/Medical/Surgery/Steps/CMSurgeryStepComponent.cs

```diff
@@ -22,4 +22,7 @@ public sealed partial class CMSurgeryStepComponent : Component
 
     [DataField]
     public ComponentRegistry? BodyRemove;
+
+    [DataField]
+    public EntProtoId? Spawn;
```

Comment:
This should be turned into a separate component effect, look at how CMSurgeryClampBleedEffectComponent, CMSurgeryStepBleedEffectComponent and CMSurgeryStepEmoteEffectComponent work for examples.